Core/include/Acts/TrackFinder/CKFSourceLinkSelector.hpp

```diff
@@ -38,8 +38,8 @@ struct GeometryCKFCriteria {
   size_t numSourcelinksCutOff = std::numeric_limits<size_t>::max();
 
   // The constructor
-  GeometryCKFCriteria(GeometryID id, double chi2, size_t num)
-      : id(id), chi2CutOff(chi2), numSourcelinksCutOff(num) {}
+  GeometryCKFCriteria(GeometryID id_, double chi2_, size_t num_)
```

Comment:
I **think** clang-tidy used to complain about this. Can we think of any alternative to the `_` suffix?

Reply:
We could use e.g. `i`, `c`, `n`.

Reply:
After a bit of reflection, I would prefer to keep the names with the `_` suffix for now. Something similar is done e.g. in the `Fatras` module. If this ends up being an issue with `clang-tidy` when can fix it ones we have `clang-tidy` working again.

Reply:
ok